packaging/debian/changelog

```diff
@@ -1,5 +1,5 @@
-flameshot (0.8.5-1) unstable; urgency=medium
```

Comment:
I guess this is a typo.

Reply:
Yeah, it should read 0.9.0-1, Fixed. Thanks